app/models/language.rb

```diff
@@ -2,6 +2,6 @@ class Language < ActiveRecord::Base
 
   has_many :campaign_page
 
-  validates_presence_of :language_code, :language_name
-  validates_uniqueness_of :language_code, :language_name
+  validates :code, :name, presence: true
```

Comment:
`language.language_code / language.lanaguage_name` were unnecessarily verbose, but perhaps it was following an AK convention? If so, can we translate it at `ak-processor`? I'm happy to make the change.